conversation/python/http/README.md

```diff
@@ -24,7 +24,34 @@ name: Install Python dependencies
 
 ```bash
 cd ./conversation
+```
+
+<details open="true">
```

Comment:
Not opposed to adding venv details but Sam does have this qs open here: https://github.com/dapr/quickstarts/pull/1208

Reply:
I'll rebase to work on top of that as it is now merged